docs/sources/administration/plugin-management/index.md

```diff
@@ -177,26 +184,28 @@ If you are a plugin developer and want to know how to sign your plugin, refer to
 | Signature status   | Description                                                                     |
 | ------------------ | ------------------------------------------------------------------------------- |
 | Core               | Core plugin built into Grafana.                                                 |
-| Invalid signature  | The plugin has a invalid signature.                                             |
+| Invalid signature  | The plugin has an invalid signature.                                             |
 | Modified signature | The plugin has changed since it was signed. This may indicate malicious intent. |
 | Unsigned           | The plugin is not signed.                                                       |
 | Signed             | The plugin signature was successfully verified.                                 |
 
 ### Plugin signature levels
```

Comment:
In the Developer Portal, we have removed a similar section and instead link directly to the [official definitions][(https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins](https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins)) provided by Grafana Legal. This eliminates inconsistencies and improves maintainability. Would that work here?

Reply:
These are different, its more about the signature check than the signature type